tracer/src/Datadog.Trace.Tools.Runner/RunCommand.cs

```diff
@@ -4,39 +4,48 @@
 // </copyright>
 
 using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
 using System.Linq;
 using Spectre.Console;
-using Spectre.Console.Cli;
 
 namespace Datadog.Trace.Tools.Runner
 {
-    internal class RunCommand : Command<RunSettings>
+    internal class RunCommand : CommandWithExamples
     {
         private readonly ApplicationContext _applicationContext;
+        private readonly RunSettings _runSettings;
 
         public RunCommand(ApplicationContext applicationContext)
+            : base("run", "Run a command with the Datadog tracer enabled")
         {
             _applicationContext = applicationContext;
+            _runSettings = new RunSettings(this);
+
+            AddExample("dd-trace run -- dotnet myApp.dll");
```

Comment:
total nit, but I saw a .NET FX person confused saying "I don't have the dotnet command" so maybe add this example too? 

```suggestion
            AddExample("dd-trace run -- dotnet myApp.dll");
            AddExample("dd-trace run -- MyApp.exe");
```